### PR TITLE
MRG: added commas for easier reading

### PR DESCRIPTION
**Pull request by ARudiuk**

### CODE REVIEW

**Comment by agramfort** [comment]
any objection from other native english speakers?

File: doc/cite.rst [hunk 1]
```diff
@@ -3,7 +3,7 @@
 How to cite MNE
 ---------------
 
-If you use in your research the implementations provided by the MNE software you should cite:
+If you use, in your research, the implementations provided by the MNE software you should cite:
```
Comment (jona-sassenhagen) [inline doc/cite.rst]: Not native either, but I would write:
`If you use the implementations provided by the MNE software in your research, you should cite:`

Comment (agramfort) [inline doc/cite.rst]: I like this better.

Comment (Eric89GXL) [inline doc/cite.rst]: agreed

**Comment by agramfort** [comment]
@Eric89GXL do it in master if you're on it.

**Comment by Eric89GXL** [comment]
Done

File: doc/cite.rst (hunk 1 above)
Comment (mmagnuski) [inline doc/cite.rst]: +1 for Jona's proposition

**Pull request closed without merging**